generators/repo/src/common/util.ts

```diff
@@ -66,6 +66,17 @@ export const cleanDirectoryPath = async (directoryPath: string, cleanGit: boolea
     await del(patterns, { cwd: directoryPath, force: true, dot: true });
 }
 
+export const ensureRelativeBasePath  = (input : string) => {
+    const basePath =`.${path.sep}`
+    if(!input.startsWith(basePath) && !input.startsWith(`.${basePath}`)) {
+        input = `${basePath}${input}`
+    }
+    else{
+        console.warn(chalk.yellowBright(` - ${input} already contains a relative base path.`));
```

Comment:
nit: May want to only write out if `--debug` flag is set?

Reply:
Thanks for the feedback, I kept the logic similar to the original design which was to keep a basic output message with the change to a file, but if --debug is set it displays a lot more details about individual substitutions in a file.